config/init.sh

```diff
@@ -68,5 +67,19 @@ envsubst '$SERVER_NAME $CATEGORY'< ./nginx_subsite.template > /etc/nginx/sites-e
 export CATEGORY=olc
 envsubst < ./env.template > /var/www/country-tz/.env
 
+#country-guinea
+export SERVER_NAME="guinea.resourcecontracts.org *.guinea.resourcecontracts.org"
+export TRACKING_ID=${GN_TRACKING_ID}
+export DB_DATABASE=${COUNTRY_DB_DATABASE}
+export CONTACT_MAIL=${GN_CONTACT_MAIL}
+export ELASTIC_SEARCH_HOST=${GN_ELASTIC_SEARCH_HOST}
```

Comment:
@manishgs where should we point this to? index.resourcecontracts.org?

Reply:
Use production API (index.resourcecontracts.org) for guinea subsite on both environment.